### PR TITLE
Update for rails 6

### PR DESCRIPTION
**Pull request by jsundt**
This updates the svg inlining script with a few old patch versions, and ensures bundler doesn't complain in rails 6.

### CODE REVIEW

File: vendor/assets/javascripts/svg_inliner/svg4everybody.js [hunk 1]
```diff
@@ -18,7 +21,7 @@
                 fragment.appendChild(clone.firstChild);
             }
             // append the fragment into the svg
-            svg.appendChild(fragment);
+            parent.appendChild(fragment);
```
Comment (kaoru) [inline vendor/assets/javascripts/svg_inliner/svg4everybody.js:24]: Is this comment above this still accurate?

File: vendor/assets/javascripts/svg_inliner/svg4everybody.js [hunk 2]
```diff
@@ -29,15 +32,15 @@
...
                 // ensure the cached html document based on the xhr response
-                cachedDocument || (cachedDocument = xhr._cachedDocument = document.implementation.createHTMLDocument(""),
+                cachedDocument || (cachedDocument = xhr._cachedDocument = document.implementation.createHTMLDocument(""), 
                 cachedDocument.body.innerHTML = xhr.responseText, xhr._cachedTarget = {}), // clear the xhr embeds list and embed each item
                 xhr._embeds.splice(0).map(function(item) {
                     // get the cached target
                     var target = xhr._cachedTarget[item.id];
                     // ensure the cached target
-                    target || (target = xhr._cachedTarget[item.id] = cachedDocument.getElementById(item.id)),
+                    target || (target = xhr._cachedTarget[item.id] = cachedDocument.getElementById(item.id)),
```
Comment (kaoru) [inline vendor/assets/javascripts/svg_inliner/svg4everybody.js:41]: Random trailing whitespace being added here

**Review by kaoru — APPROVED** [review]
I'm not sure all these changes belong in a single commit ⚠️ I'm also not gonna say I understand all the polyfill/parent/numberOfSvgElementsToBypass stuff. But I like that it looks like you got to kill a `/svg/i.test(svg.nodeName)` in favour of some real URL parsing by the looks of things 😅 Approved! ✅

**Comment by jsundt** [comment]
I've split it into two commits :)

The polyfill stuff is all vendor, I've had a quick look over and it seems fine. I left the comments and whitespace as provided.

**Comment by kaoru** [comment]
Thanks @jsundt, much clearer 😄

**Pull request merged**